.travis.yml

```diff
@@ -13,35 +13,40 @@ sudo: false
 cache:
   directories:
     - $HOME/.composer/cache
- 
+
 env:
-  - SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+  global:
+    - SYMFONY_DEPRECATIONS_HELPER=weak
+  matrix:
+    - SYMFONY_VERSION=2.7.*
 
 matrix:
   include:
     - php: 5.3
       env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.3.* SYMFONY_DEPRECATIONS_HELPER=weak
+      env: SYMFONY_VERSION=2.3.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.*
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=strict
     - php: 5.6
-      env: SYMFONY_VERSION=3.0.*
+      env: SYMFONY_VERSION=3.0.* SYMFONY_DEPRECATIONS_HELPER=strict
   allow_failures:
-    - env: SYMFONY_VERSION=2.8.*
-    - env: SYMFONY_VERSION=3.0.*
+    - env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=strict
+    - env: SYMFONY_VERSION=3.0.* SYMFONY_DEPRECATIONS_HELPER=strict
   fast_finish: true
 
 before_install:
   - if [[ "$TRAVIS_PHP_VERSION" != "hhvm" ]]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
   - composer self-update
   - composer require symfony/symfony:${SYMFONY_VERSION} --no-update
-  
-install: composer update --prefer-source $COMPOSER_FLAGS
 
-before_script: vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh
+install: composer update --prefer-dist $COMPOSER_FLAGS
+
+before_script:
+  - vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh
+  - php vendor/symfony-cmf/testing/bin/console doctrine:phpcr:fixtures:load --fixtures=Tests/Resources/DataFixtures --no-interaction
 
-script: phpunit --coverage-text
+script: phpunit
```

Comment:
Removed the coverage as it was useless with the parallel testing. I didn't add it back after removing parallel testing, because it reduces testing performance and I never looked at it.
I'm open to add it back /cc @dbu @lsmith77 @dantleech